app/assets/javascripts/alchemy/alchemy.node_tree.js

```diff
@@ -0,0 +1,75 @@
+Alchemy.NodeTree = {
+  onFinishDragging: function (evt) {
+    var url = Alchemy.routes.move_api_node_path(evt.item.dataset.id)
+    var xhr = Alchemy.xhr('PATCH', url)
```

Comment:
Thanks. I see us moving the callback and response parsing into that helper as well. We could also do that in a subsequent PR.